src/components/payroll/PayrollStatusPicker.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { ConstantBasedPicker } from '@openimis/fe-core';
+
+import { PAYROLL_STATUS_LIST } from '../../constants';
+
+function PayrollStatusPicker(props) {
+  const {
+    required, withNull, readOnly, onChange, value, nullLabel, withLabel,
+  } = props;
```

Comment:
This seems to be quite long. Maybe better aproach would be something like that: 
`
function PayrollStatusPicker({
required,
withNull,
readOnly,
onChange,
value,
nullLabel,
withLabel,
}) {
return (...)
}
`

Reply:
I changed this too in dependent PR